airbyte-webapp-e2e-tests/cypress/integration/settings.spec.ts

```diff
@@ -0,0 +1,33 @@
+import { initialSetupCompleted } from "commands/workspaces";
+import {
+  clickUpgradeAllButton,
+  editVersionByConnectorName,
+  goToSettingsDestinationTab,
+  goToSettingsSourcesTab,
+} from "pages/settingsPage";
+
+describe("Source settings main actions", () => {
+  beforeEach(() => {
+    initialSetupCompleted();
+    goToSettingsSourcesTab();
+  });
+  it("downgrades a source version and updates it through the input", () => {
+    editVersionByConnectorName("Postgres", "0.1.0");
+    editVersionByConnectorName("Postgres", "0.1.1");
+  });
+  it("downgrades a source version and updates it through 'upgrade all' button", () => {
```

Comment:
Not sure if it is intentional, but an "Upgrade all" button is only present on the `sources` tab, hence why only this describe block has this test.